algorithms/ex_1.rb

```diff
@@ -0,0 +1,33 @@
+def solution(n)
+ 
+  counter = 0
+  max_value = 0
+
+  if n == 1
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression (ie. `return counter if n == 1`)